spring-cloud-gcp-data-spanner/README.adoc

```diff
@@ -0,0 +1,72 @@
+:ext-relative: adoc
+
+= image:spanner-logo.png[50,50] Spring Data Spanner
+
+Provides a http://projects.spring.io/spring-data/[Spring Data Module] for https://cloud.google.com/spanner/[Google Cloud Spanner].
+
+Maven coordinates for this module only, using Spring Cloud GCP BOM:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-gcp-data-spanner</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-data-spanner'
+}
+----
+
+With our link:../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Spring Boot Starter for Spring Data Spanner], you can leverage our recommended auto-configuration setup. (see https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-starter[Spring Boot Starters] for more details on what a starter is).
```

Comment:
Remove the parenthesis. Anyone who doesn't know what a Spring Boot starter is can just Google it on their own.

Reply:
I don't agree, I like us to be more helpful than that.

Reply:
By that logic, you'd have to add it to every place in the refdoc where we mention Spring Boot starters. And explain every term that you think users won't understand.